### PR TITLE
Natively handles bz2 compressed model files.

### PR DESCRIPTION
**Pull request by halfak**
This will allow us to store compressed model files and safely use them.

### CODE REVIEW

**Comment by Ladsgroup** [comment]
We have an unhappy travis here, do flake8 :D

File: revscoring/scoring/models/model.py [hunk 1]
```diff
@@ -156,7 +164,7 @@ def __init__(self, *args, scale=False, center=False, **kwargs):
         })
 
     def train(self, values_labels):
-        """
+        """models/enwiki.draft_quality.gradient_boosting.model
```
Comment (Ladsgroup) [inline revscoring/scoring/models/model.py]: This seems like an error

**Comment by halfak** [comment]
Thanks!  Got it.

**Comment by halfak** [comment]
Self merging with cleaned flake8

**Pull request merged**